ros2bag/ros2bag/verb/play.py

```diff
@@ -56,6 +56,10 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             '--remap', '-m', default='', nargs='+',
             help='list of topics to be remapped: in the form '
                  '"old_topic1:=new_topic1 old_topic2:=new_topic2 etc." ')
+        parser.add_argument(
+            '--ignore-unknown-types', action='store_true',
+            help='enables playing back bag files with unknown messages, skipping the message '
+                 'types that are unknown.')
```

Comment:
I am actually not sure if that command line option is really needed. I think it makes sense to simply skip over the non-playable messages and issue a warning with it.